app/src/main/java/com/readrops/app/more/preferences/PreferencesScreenModel.kt

```diff
@@ -56,16 +61,21 @@ class PreferencesScreenModel(
                         customShareIntentTpl = (list[9] as String) to customShareIntentTpl,
                         swipeToLeft = (list[10] as String) to swipeToLeft,
                         swipeToRight = (list[11] as String) to swipeToRight,
-                        exampleItem = if (database.itemDao().count() > 0) {
-                            database.itemDao().selectFirst()
-                        } else {
```

Comment:
I'm just removing this. Event though using real data makes for a better UX generally speaking, querying an `ItemWithFeed` is just too difficult from here.